api/app/signals/apps/search/settings.py

```diff
@@ -0,0 +1,57 @@
+from django.conf import settings
+from django.test.signals import setting_changed
+
+DEFAULTS = dict(
+    PAGE_SIZE=100,
+    CONNECTION=dict(
+        URL='http://127.0.0.1:9200',
+        INDEX_NAME='sia_signals',
+    ),
+)
+
+
+class APISettings(object):
```

Comment:
Double check: this is up-to-date with the `APISettings` object that we reviewed/adapted before?

Reply:
Yes it is :)